packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.html

```diff
@@ -124,12 +124,6 @@
                                     data-ng-if="$ctrl.server.os"
                                     data-ng-bind="$ctrl.server.os === 'byoi_32' ? ('server_configuration_distribution_byoi' | translate) : $ctrl.server.os"
                                 ></span>
-                                <span
-                                    class="oui-badge oui-badge_warning"
-                                    data-ng-if="!$ctrl.server.os"
```

Comment:
related to your Manager ticket, i guess you have to do the restriction only for case where `osName === none_64`.